system_modules/naprender/src/shader.cpp

```diff
@@ -266,7 +266,10 @@ class AllowIncluder : public glslang::TShader::Includer
 
 		auto absolute_path = nap::utility::findFileInDirectories(headerName, { shader_search_paths });
 		if (absolute_path.empty())
-			return nullptr;
```

Comment:
Why assert instead of issuing a warning? Asserts are for programmer errors that should not (never) occur. If the call is allowed to fail and return nothing an assert is the probably not the right choice here.

Reply:
Yes agreed